build.gradle.kts

```diff
@@ -27,10 +27,77 @@ val collectCatalogTypings by tasks.registering(Sync::class) {
 
 val collectExternalTypings by tasks.registering(Sync::class) {
     mapOf(
-        "Vampire_setup-wsl_v4.yml" to "Vampire/setup-wsl/refs/heads/v4/action-types.yml",
-        "Vampire_setup-wsl_master.yml" to "Vampire/setup-wsl/refs/heads/master/action-types.yml",
-        "typesafegithub_github-actions-typing_v1.yml" to "typesafegithub/github-actions-typing/refs/heads/v1/action-types.yml",
+        "abhirajputnikku7_android-emulator-runner_main.yml" to "abhirajputnikku7/android-emulator-runner/refs/heads/main/action-types.yml",
+        "achrinza_setup-db2_main.yaml" to "achrinza/setup-db2/refs/heads/main/action-types.yaml",
+        "actions-marketplace-validations_BiTaksi_mongodb-github-action_main.yml" to "actions-marketplace-validations/BiTaksi_mongodb-github-action/refs/heads/main/action-types.yml",
```

Comment:
Let's remove all forks that belong to https://github.com/actions-marketplace-validations - it's a weird experiment that introduces noise to the usage of this action. I don't want these forks to be used in the tests.